docs/AboutUs.md

```diff
@@ -59,3 +59,14 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 * Role: Developer
 * Responsibilities: UI
+
+
+### Christopher Tan
+
+<img src="images/chrisjohntan.png" width="200px">
+
+[[github](http://github.com/chrisjohntan)]
+[[portfolio](team/chrisjohntan.md)]
```

Comment:
I like that you specified the template portfolio md file to your Github username to deconflict!